target_stitch/__init__.py

```diff
@@ -260,7 +260,7 @@ def handle_batch(self, messages, schema, key_names, bookmark_names=None): # pyli
                                         i, k))
                 except Exception as e:
                     raise TargetStitchException(
-                        'Record does not pass schema validation: {}'.format(e))
+                        'Record does not pass schema validation: {}, {}'.format(e, message))
```

Comment:
Let's log the message at DEBUG level instead of embedding it in the exception. We try never to have client data present in our logs and we guarantee that we won't save DEBUG and up logs.